ui/app/components/secure-variable-form.js

```diff
@@ -31,6 +32,23 @@ export default class SecureVariableFormComponent extends Component {
    * @type {DuplicatePathWarning}
    */
   @tracked duplicatePathWarning = null;
```

Comment:
There is a `validatePath()` action that should be updated here (I think! @tgross  / @angrycub please correct me if this is an incorrect assumption): If I have a variable at `my-vars/a` on the `default` namespace, but try to create a new one at `my-vars/a` on the `my-other-namespace` namespace, we currently throw a duplicate error. I think this is probably supposed to be allowed, as confusing as the circumstance might be to the end-user.